# HG changeset patch
# User jwe@segfault.lan
# Date 1201918354 18000
#      Fri Feb 01 21:12:34 2008 -0500
# Node ID 3f89170d124701fe140e511113550b1115fb68fa
# Parent  1c65a8e44ef95fd4b2b11eb19071e3ecbbe77b76
Added tag ss-2-1-59 for changeset 1c65a8e44ef9

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -1,8 +1,9 @@
 d56e0d75e7c9760f2a3e750c2d71a5c52bb76b98 ss-2-1-45
 dece11da64ed86ac2347f48ffa13fd046bef7de3 ss-2-1-52
 e759d01692dbb69ff28b6d70940239d93ff77a9d ss-2-1-53
 adf8d68d71438b8b394cbbce59e817a557db5ecd ss-2-1-54
 6279796dc2b9ee28e0624b4b91bba1eec6ed292d ss-2-1-55
 d0aa1a59b73b879fe18eafb8df54e451f870058c ss-2-1-56
 c1cdf2a32cdbf4e0d68aead4bfa597106e844989 ss-2-1-57
 ce01dbd7e026c095d9e50beac529e8e955a303f8 ss-2-1-58
+1c65a8e44ef95fd4b2b11eb19071e3ecbbe77b76 ss-2-1-59
